components/TextField/TextField.stories.tsx

```diff
@@ -70,6 +70,40 @@ export const ReadOnly = Basic.bind({});
 ReadOnly.args = { id: 'readonly', readOnly: true };
 ignoreArgType('id', ReadOnly);
 
+const StyledCopyIcon = styled(CopyIcon, {
+  '@hover': {
+    '&:hover': {
+      cursor: 'pointer'
+    }
+  }
+});
+
+export const ReadOnlyCopy: ComponentStory<typeof TextFieldForStory> = (args) => {
+  const toCopy = 'Text to copy';
+
+  const onCopy = useCallback(
+    async () => {
+      await navigator.clipboard.writeText(toCopy)
+    },
```

Comment:
[optional]
I think this could be simplified to just:

```suggestion
    () => navigator.clipboard.writeText(toCopy),
```

Reply:
Yeap looks good to me 👍